src/status_im2/contexts/chat/lightbox/view.cljs

```diff
@@ -73,131 +83,167 @@
+                          #(reset! set-full-height? true)
+                          (anim/animate (if x? pan-x pan-y) 0)
+                          (anim/animate opacity 1)
+                          (anim/animate layout 0)))))))
 
 (defn lightbox
   []
   [:f>
    (fn []
-     (let [{:keys [messages index]} (rf/sub [:get-screen-params])
-           atoms                    {:flat-list-ref      (atom nil)
-                                     :small-list-ref     (atom nil)
-                                     :scroll-index-lock? (atom true)
-                                     :insets-atom        (atom nil)}
+     ;; we get `insets` from `screen-params` because trying to consume it from
+     ;; lightbox screen causes lots of problems
+     (let [{:keys [messages index insets]} (rf/sub [:get-screen-params])
+           atoms                           {:flat-list-ref      (atom nil)
+                                            :small-list-ref     (atom nil)
+                                            :scroll-index-lock? (atom true)}
            ;; The initial value of data is the image that was pressed (and not the whole album) in order
            ;; for the transition animation to execute properly, otherwise it would animate towards
            ;; outside the screen (even if we have `initialScrollIndex` set).
-           data                     (reagent/atom [(nth messages index)])
-           scroll-index             (reagent/atom index)
-           transparent?             (reagent/atom false)
-           window                   (rf/sub [:dimensions/window])
-           animations               {:background-color (anim/use-val "rgba(0,0,0,0)")
-                                     :border           (anim/use-val (if platform/ios? 0 12))
-                                     :opacity          (anim/use-val 0)
-                                     :rotate           (anim/use-val "0deg")
-                                     :top-layout       (anim/use-val -10)
-                                     :bottom-layout    (anim/use-val 10)
-                                     :top-view-y       (anim/use-val 0)
-                                     :top-view-x       (anim/use-val 0)
-                                     :top-view-width   (anim/use-val (:width window))
-                                     :top-view-bg      (anim/use-val colors/neutral-100-opa-0)}
-
-           callback                 (fn [e]
-                                      (on-viewable-items-changed e scroll-index atoms))
-           insets-ios               (when platform/ios? (safe-area/use-safe-area))]
+           data                            (reagent/atom [(nth messages index)])
+           scroll-index                    (reagent/atom index)
+           transparent?                    (reagent/atom false)
+           set-full-height?                (reagent/atom false)
+           window-width                    (rf/sub [:dimensions/window-width])
```

Comment:
I believe you could subscribe to just one subscription `:dimensions/window` and destructure the `:width` and `:height` keys. Just slightly less overhead than using two re-frame reactions.